Assets/unZENity-VR/Scripts/Settings/SettingsManager.cs

```diff
@@ -65,5 +72,28 @@ private void CheckIfGothicIDirectoryExists()
                     $"Please put in the right absolute path to your local installation.");
         }
 
+        /// <summary>
+        /// Import the settings file from streamingAssetPath to persistentDataPath.
+        /// Since the settings file is in streamingAssetPath, we need to use UnityWebRequest to move it so we can have access to it
+        /// as detailed here https://docs.unity3d.com/ScriptReference/Application-streamingAssetsPath.html
+        /// </summary>
+        private void Importer()
```

Comment:
Can you rename this method, please. I think of something which tells us "It's cloning the GameSettings.json file to be altered for next game start".

How about sth. like _CopyGameSettingsForAndroidBuild()_?

Reply:
I'll rename it 👍